src/Gui/PreferencePacks/FreeCAD Classic/FreeCAD Classic.cfg

```diff
@@ -9,6 +9,9 @@
         </FCParamGroup>
         <FCParamGroup Name="View">
           <FCUInt Name="HighlightColor" Value="3789624575"/>
+          <FCUInt Name="AxisYColor" Value="869020416"/>
+          <FCUInt Name="AxisZColor" Value="859032576"/>
+          <FCUInt Name="HighlightColor" Value="481107199"/>
```

Comment:
```suggestion
          <FCUInt Name="AxisXColor" Value="3425907456"/>
          <FCUInt Name="AxisYColor" Value="869020416"/>
          <FCUInt Name="AxisZColor" Value="859032576"/>
```

Reply:
Can the highlight color not be coupled to one of the three(!) accent colors?

Reply:
AFAIK the highlight color has no relation to this PR so that discussion should happen elsewhere.